codersdk/deployment.go

```diff
@@ -52,6 +52,7 @@ const (
 	FeatureWorkspaceBatchActions      FeatureName = "workspace_batch_actions"
 	FeatureAccessControl              FeatureName = "access_control"
 	FeatureOAuth2Provider             FeatureName = "oauth2_provider"
+	FeatureControlSharedPorts         FeatureName = "control_shared_ports"
```

Comment:
`workspace_custom_shared_ports` would be a better feature name IMO

Reply:
I want to describe that the feature is actually just the ability to restrict sharing ports. Hence "control" because I didn't want to use "restrict". What does the "custom" mean here?

Reply:
What about `restrict_shared_ports`?